ConsumerApi.Sdk/Endpoints/Challenges/ChallengesEndpoint.cs

```diff
@@ -6,8 +6,23 @@ namespace Backbone.ConsumerApi.Sdk.Endpoints.Challenges;
 
 public class ChallengesEndpoint(EndpointClient client) : ConsumerApiEndpoint(client)
 {
-    public async Task<ApiResponse<Challenge>> CreateChallenge() => await _client.Post<Challenge>($"api/{API_VERSION}/Challenges");
-    public async Task<ApiResponse<Challenge>> CreateChallengeUnauthenticated() => await _client.PostUnauthenticated<Challenge>($"api/{API_VERSION}/Challenges");
+    public async Task<ApiResponse<Challenge>> CreateChallenge()
+    {
+        return await _client.Post<Challenge>($"api/{API_VERSION}/Challenges");
+    }
 
-    public async Task<ApiResponse<Challenge>> GetChallenge(string id) => await _client.Get<Challenge>($"api/{API_VERSION}/Challenges/{id}");
+    public async Task<ApiResponse<Challenge>> CreateChallengeUnauthenticated()
+    {
+        return await _client.PostUnauthenticated<Challenge>($"api/{API_VERSION}/Challenges");
+    }
+
+    public async Task<ApiResponse<Challenge>> GetChallenge(string id)
+    {
+        return await _client.Get<Challenge>($"api/{API_VERSION}/Challenges/{id}");
+    }
+
+    public async Task<ApiResponse<Challenge>> GetChallengeUnauthenticated(string id)
+    {
+        return await _client.GetUnauthenticated<Challenge>($"api/{API_VERSION}/Challenges/{id}");
+    }
```

Comment:
It is impossible to get a challenge without authentication

Reply:
https://github.com/nmshd/backbone/blob/NMSHDB-83-create-identities-via-sdk/ConsumerApi.Tests.Integration/Features/Challenges/%7Bid%7D/GET.feature#L13-L17

In that case, should we ditch this scenario?

Reply:
Yeah, I would only test scenarios that are possible with the sdk. At least for now.